frameworks/kafka/docs/configure.md

```diff
@@ -27,82 +27,53 @@ The Kafka scheduler runs as a Marathon process and can be reconfigured by changi
 
-    $ curl -H "Authorization: token=$AUTH_TOKEN" "$DCOS_URI/service/kafka/v1/plan"
-    GET $DCOS_URI/service/kafka/v1/plan HTTP/1.1
+    curl -H "Authorization: token=$auth_token" "<dcos_url>/service/kafka/v1/plan"
 
     {
-      "phases": [
-        {
-          "id": "1915bcad-1235-400f-8406-4ac7555a7d34",
-          "name": "Reconciliation",
-          "steps": [
-            {
-              "id": "9854a67d-7803-46d0-b278-402785fe3199",
-              "status": "COMPLETE",
-              "name": "Reconciliation",
-              "message": "Reconciliation complete"
-            }
-          ],
-          "status": "COMPLETE"
-        },
+      "phases" : [
         {
-          "id": "3e72c258-1ead-465f-871e-2a305d29124c",
-          "name": "Update to: 329ef254-7331-48dc-a476-8a0e45752871",
-          "steps": [
+          "id" : "b6180a4e-b25f-4307-8855-0b37d671fd46",
+          "name" : "Deployment",
+          "steps" : [
             {
-              "id": "ebf4cb02-1011-452a-897a-8c4083188bb2",
-              "status": "COMPLETE",
-              "name": "broker-0",
-              "message": "Broker-0 is COMPLETE"
+              "id" : "258f19a4-d6bc-4ff1-8685-f314924884a1",
+              "status" : "COMPLETE",
+              "name" : "kafka-0:[broker]",
+              "message" : "com.mesosphere.sdk.scheduler.plan.DeploymentStep: 'kafka-0:[broker] [258f19a4-d6bc-4ff1-8685-f314924884a1]' has status: 'COMPLETE'."
             },
             {
-              "id": "ff9e74a7-04fd-45b7-b44c-00467aaacd5b",
-              "status": "COMPLETE",
-              "name": "broker-1",
-              "message": "Broker-1 is COMPLETE"
+              "id" : "e59fb2a9-22e2-4900-89e3-bda24041639f",
+              "status" : "COMPLETE",
+              "name" : "kafka-1:[broker]",
+              "message" : "com.mesosphere.sdk.scheduler.plan.DeploymentStep: 'kafka-1:[broker] [e59fb2a9-22e2-4900-89e3-bda24041639f]' has status: 'COMPLETE'."
             },
             {
-              "id": "a2ba3969-cb18-4a05-abd0-4186afe0f840",
-              "status": "COMPLETE",
-              "name": "broker-2",
-              "message": "Broker-2 is COMPLETE"
+              "id" : "0b5a5048-fd3a-4b2c-a9b5-746045176d29",
+              "status" : "COMPLETE",
+              "name" : "kafka-2:[broker]",
+              "message" : "com.mesosphere.sdk.scheduler.plan.DeploymentStep: 'kafka-2:[broker] [0b5a5048-fd3a-4b2c-a9b5-746045176d29]' has status: 'COMPLETE'."
             }
           ],
-          "status": "COMPLETE"
-        }
-      ],
-      "errors": [],
-      "status": "COMPLETE"
-    }
-
-
-
+        "status" : "COMPLETE"
+      }
+    ],
+    "errors" : [ ],
+    "status" : "COMPLETE"
+  }
+
+<!-- need to update this with current information for different deployments
 When using the `STAGE` deployment strategy, an update plan will initially pause without doing any update to ensure the plan is correct. It will look like this:
```

Comment:
phase strategy is canary

I think it is much easier to explain with yml file:
            dest: "{{KAFKA_VERSION_PATH}}/config/server.properties"
plans:
  deploy:
    strategy: serial
    phases:
      Deployment:
        strategy: {{DEPLOY_STRATEGY}}
        pod: kafka